mavros_extras/src/plugins/mocap_fake_gps.cpp

```diff
@@ -142,48 +142,40 @@ class MocapFakeGPSPlugin : public plugin::PluginBase
 		old_t = trans->header.stamp.toSec();
 
 		//calculate velocities
-		double vn = 100 * dn / dt; 		//[cm/s]
-		double ve = 100 * de / dt; 		//[cm/s]
-		double vd = - 100 * dz / dt; 	//[cm/s]
+		double vn = 100 * dn / dt;	//[cm/s]
+		double ve = 100 * de / dt;	//[cm/s]
+		double vd = -100 * dz / dt;	//[cm/s]
 
 		//calculate course over ground
 		double _cog_rad;
 
-		if (vn == 0 && ve == 0) 
+		if (vn == 0 && ve == 0)
 		{
 			_cog_rad = 0;
 		}
-		else if (vn >= 0 && ve < 0) 
+		else if (vn >= 0 && ve < 0)
 		{
 			_cog_rad = M_PI * 5 / 2 - atan2(vn,ve);
-		} 
-		else 
+		}
+		else
 		{
 			_cog_rad = M_PI / 2 - atan2(vn,ve);
 		}
 
 		double _cog_deg = _cog_rad * 180 / M_PI;
-		uint8_t fix_type = 3; // 3D
-		int32_t lat = (lat_rad * 180 / M_PI) * 10000000; // [degrees * 1E7]
-		int32_t lon = (lon_rad * 180 / M_PI) * 10000000; // [degrees * 1E7]
-		int32_t alt = (408 + z) * 1000; 		// [m * 1000] AMSL
-		uint16_t vel = sqrt(vn*vn + ve*ve); // [cm/s]
-		uint16_t cog = _cog_deg * 100; 			// [degrees * 100]
-		uint8_t satelites_visible = 5;
+		int32_t lat = (lat_rad * 180 / M_PI) * 10000000;// [degrees * 1E7]
+		int32_t lon = (lon_rad * 180 / M_PI) * 10000000;// [degrees * 1E7]
+		int32_t alt = (408 + z) * 1000;			// [m * 1000] AMSL
+		uint16_t vel = sqrt(vn * vn + ve * ve);		// [cm/s]
+		uint16_t cog = _cog_deg * 100;			// [degrees * 100]
 
 		gps_pose_send(trans->header.stamp.toNSec() / 1000,
-									lat, lon, alt,
-									gps_eph, gps_epv,
-									vel,
-									vn, ve, vd,
-									cog, fix_type,
-									satelites_visible);
+					lat, lon, alt,
+					vel, vn, ve, vd, cog);
```

Comment:
Because a long sequence of expressions is less clear then a sequence of names, you don't immediately know what the i-th argument represents. Maybe filling in the message directly is actually better than having a function with so many arguments.